Testing/VelaptorTests/Content/TexturePathResolverTests.cs

```diff
@@ -95,22 +96,21 @@ public void ResolveFilePath_WhenInvoked_ResolvesFilepath(string contentName)
         // Arrange
         var mockDirectory = new Mock<IDirectory>();
         mockDirectory.Setup(m => m.GetFiles(this.atlasContentDir, "*.png"))
-            .Returns(() =>
-            {
-                return new[]
+            .Returns(
+                new[]
                 {
                     $"{this.atlasContentDir}/other-file.png",
                     this.contentFilePath,
-                };
-            });
+                }
+            );
```

Comment:
Please apply the same style changes to this test arrangement code as in the previous comment for the `ResolveFilePath_WhenContentItemDoesNotExist_ThrowsException()` unit test method.

```suggestion
        mockDirectory.Setup(m => m.GetFiles(this.atlasContentDir, "*.png"))
            .Returns(new[]
                {
                    $"{this.atlasContentDir}/other-file.png",
                    this.contentFilePath,
                });
```